Lib/fontTools/subset/cff.py

```diff
@@ -470,6 +498,7 @@ def desubroutinize(self):
 				del pd.Subrs
 			if 'Subrs' in pd.rawDict:
 				del pd.rawDict['Subrs']
+	# as well as the global subrs
```

Comment:
How is this change relevant to the issue being fixed?

Reply:
I asked Read to apply that patch since he was working on related issues with the —desubroutinize option:
https://github.com/fonttools/fonttools/pull/1499#issuecomment-463309891